src/plots/gl2d/scene2d.js

```diff
@@ -268,6 +269,26 @@ proto.updateFx = function(options) {
     fullLayout.hovermode = options.hovermode;
 };
 
+var relayoutCallback = function(scene) {
+
+    var update = {},
+        xrange = scene.xaxis.range,
+        yrange = scene.yaxis.range;
+
+    // Update the layout on the DIV
+    scene.graphDiv.layout.xaxis.range = xrange.slice(0);
+    scene.graphDiv.layout.yaxis.range = yrange.slice(0);
+
+    // Make a meaningful value to be passed on to the possible 'plotly_relayout' subscriber(s)
+    update[scene.id] = { // scene.camera has no many useful projection or scale information
+        lastInputTime: scene.camera.lastInputTime, // helps determine which one is the latest input (if async)
+        xrange: xrange.slice(0),
+        yrange: yrange.slice(0)
+    };
```

Comment:
In gl2d, `scene` is purely internal. I'd vote for something like:

```js
update = {
  lastInputTime: scene.camera.lastInputTime  // great idea here!
};
update[xaxis._name] = xrange.slice();
update[yaxis._name] = yrange.slice();
```

where `?axis.name` will be `xaxis`, `xaxis2`, `yaxis`, `yaxis2` ...